btree/btree.c

```diff
@@ -3758,13 +3766,18 @@ static int64_t btree_get_tick(struct m0_sm_op *smop)
 
 		bop->bo_cb.c_act(&bop->bo_cb, &s.s_rec);
 
-		lock_op_unlock(tree);
+		lock_op_unlock(tree, oi);
 		return P_CLEANUP;
```

Comment:
it should be `return m0_sm_op_sub(&bop->bo_op, P_CLEANUP, P_FINI);`

Reply:
done